### PR TITLE
Pr for history page

### PR DESCRIPTION
**Pull request by narharikale**
Added history Page

### CODE REVIEW

File: src/frontend/screen/SingleVideo/SingleVideo.jsx [hunk 1]
```diff
@@ -49,6 +49,11 @@ function SingleVideo(){
         setLikesData(data.likes);
     }
 
+    const addToHistory = async() =>  {
+        const  data  =  await combinedService("post" , "/api/user/history" , isAuth.token , video );
```
Comment (MiheerTamkhane) [inline src/frontend/screen/SingleVideo/SingleVideo.jsx:53]: Nice use of this combinedSerivce

**Review by MiheerTamkhane — APPROVED** [review]
Code is good, Readable and clean, alos DRY is used

File: src/frontend/screen/history/History.jsx [hunk 2]
```diff
@@ -1,11 +1,50 @@
...
 function History(){
-    
 
+    const { isAuth } = useAuth();
+    const { historyData , setHistoryData } = useHistory();
+    const id = historyData.length > 0 ? historyData[historyData.length-1]._id : "" ;
+
+    const removeAllHistory = async() => {
+        const { data } = await axios.delete("/api/user/history/all" , { headers: { authorization : isAuth.token }}  );
+        console.log(data);
```
Comment (rahulmistry751) [inline src/frontend/screen/history/History.jsx]: Remove console log. if there is no further use of it.

**Review by rahulmistry751 — APPROVED** [review]
Code looks clean and well structured.

**Pull request merged**